cumulusci/core/keychain/encrypted_file_project_keychain.py

```diff
@@ -95,30 +120,32 @@ def _encrypt_config(self, config):
         return base64.b64encode(iv + cipher.encryptor().update(pickled))
 
     def _decrypt_config(self, config_class, encrypted_config, extra=None, context=None):
-        if not encrypted_config:
-            if extra:
-                return config_class(None, *extra)
-            else:
-                return config_class()
-        encrypted_config = base64.b64decode(encrypted_config)
-        iv = encrypted_config[:16]
-        cipher, iv = self._get_cipher(iv)
-        pickled = cipher.decryptor().update(encrypted_config[16:])
-        try:
-            unpickled = pickle.loads(pickled, encoding="bytes")
-        except Exception:
-            raise KeychainKeyNotFound(
-                f"Unable to decrypt{' ' + context if context else ''}. "
-                "It was probably stored using a different CUMULUSCI_KEY."
-            )
-        # Convert bytes created in Python 2
-        config_dict = {}
-        for k, v in unpickled.items():
-            if isinstance(k, bytes):
-                k = k.decode("utf-8")
-            if isinstance(v, bytes):
-                v = v.decode("utf-8")
-            config_dict[k] = v
+        if self.key:
```

Comment:
I guess this is just a matter of taste but I would do:
```
if self.key is None:
    return
```

which makes it clearer that nothing more is going to happen in this case, and avoids extra indentation.

Reply:
I would do that, but we need to hit line 149 and below if they key isn't present. Maybe I can refactor it somehow so that logic at the bottom doesn't need to be in `_decrypt_config()`.